h/static/scripts/annotator/guest.coffee

```diff
@@ -71,6 +71,13 @@ module.exports = class Guest extends Annotator
     this._connectAnnotationSync(@crossframe)
     this._connectAnnotationUISync(@crossframe)
 
+    if @options.clickToClose
+      @wrapper = @element
+      .on 'click', (event) =>
+        if !@selectedTargets?.length
+          this.hideFrame()
+      this
```

Comment:
The `this` here was at the bottom of the function in order to make this interface fluent. That would have been clearer if it said `return this`.

Could you move this line to the end of the function again and write it as `return this`?